Automattic/wp-calypso
# Permafail E2E: redesign the Help Centre spec.

Fixes https://github.com/Automattic/wp-calypso/issues/78358.

## Proposed Changes

This PR rethink the Support (now renamed Help Centre) specs.

Key changes:
- remove the spec that tests various inputs. [This is better served as a component test](https://github.com/Automattic/wp-calypso/issues/77724).
- rename the specs to reference Help Center (US spelling).
- refactor the remaining spec to simulate a user looking for information on a topic.

## Testing Instructions

Ensure the following build configurations are passing:

100 iterations survived on desktop viewport:
![image](https://github.com/Automattic/wp-calypso/assets/6549265/9e0110c7-b415-4892-9053-4b850d9ec2c8)

100 iterations survived on mobile viewport:
![image](https://github.com/Automattic/wp-calypso/assets/6549265/03134225-21ac-428e-8dd4-5a1b4549eaa5)


## Pre-merge Checklist
